unlock-app/src/components/interface/locks/List/elements/LockList.tsx

```diff
@@ -103,30 +101,26 @@ export interface FavoriteLocks {
 }
 
 export const LockList = ({ owner }: LockListProps) => {
+  const config = useConfig()
```

Comment:
Hum. Ant way we can just use the import? I think the him is not the right approach

Reply:
I think so too!